.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Can you remove this file for now ? I am not completely opposed to adding it, but lets discuss in a separate PR if you think it's necessary.